.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "nanocube"]
```

Comment:
I think we forgot to move this file when we copied the directory into your GOPATH. Could you copy it over now?